### PR TITLE
Typeclass tweaks

### PR DESCRIPTION
**Pull request by Kelketek**
A couple of typeclass tweaks to be merged ahead of time for collab, but without having any collab-specific code.

### CODE REVIEW

File: evennia/typeclasses/models.py [hunk 1]
```diff
@@ -126,6 +126,31 @@ class Meta:
         return new_class
 
 
+class DbHolder(object):
```
Comment (Griatch) [inline evennia/typeclasses/models.py]: Should probably break out NdbHolder too, just for the symmetry of it then.

Comment (Kelketek) [inline evennia/typeclasses/models.py]: @Griatch Alright. I'll get that to you here in a bit, then.

Comment (Kelketek) [inline evennia/typeclasses/models.py]: @Griatch Done. NDBHolder was almost verbatim the same as DBHolder, so I consolidated them into DBHolder with an argument.

**Comment by Griatch** [comment]
Merged with rebase.

**Pull request closed without merging**